packages/protocol/contracts/governance/Governance.sol

```diff
@@ -649,10 +667,70 @@ contract Governance is IGovernance, Ownable, Initializable, UsingLockedGold, Ree
   }
 
   /**
-   * @notice Withdraws refunded Celo Gold commitments.
+   * @notice Whitelists the hash of a hotfix.
+   * @param txHash The hash of the hotfix to be whitelisted.
+   * @param validatorIndex The index in the validator set of the whitelisting validator.
+   */
+  function whitelist(bytes32 txHash, uint256 validatorIndex) external nonReentrant {
```

Comment:
it seems to me that the same validator can call this function 2n+1 times and get done with it